adoptingerlang.org

```diff
@@ -3997,14 +3997,85 @@ No business works only with code; it is way more frequent to hear about projects
+
+This is, ultimately, not too different from regular development, but you'll be headed that way willingly. As such, you may want to structure things a bit differently. Rather than picking one expert and your enthusiastic folks, you will want to explode the requirements and roles across everyone, to try and get the dynamics than an expert would have facilitated.
+
+Identify people who have prior knowledge of functional languages; they won't know everything, but they already have a head start. Also identify some self-directed learners, those who enjoy sitting down and digging into something. Their job will be to read books and figure things out a few weeks ahead of everyone else; they're quire literally the piano teacher only ahead of you by a couple of weeks. You'll want to find someone who enjoys teaching, writing documentation, or preparing demos. That's often pretty hard, so if you can't find anyone, assign a rotating schedule to that responsibility. Put them in charge of doing broader information dissemination, maintaining docs, and a quick "getting started" track to what you do in your organisation.
+
+Have everyone do bits of development and getting used to the feel of the language, getting involved with things in the community, and leverage the code already existing out there. Here are a few ideas:
+
+- look into existing code out there, and do a show-and-tell of various libraries and their approaches; dig to figure out how they are structured.
+- if you can't figure out why an open-source library does what it does, feel free to ask their authors in an issue, or people on slack, mailing lists, or a forum. Get the real answer, and show it out there
+- do code reviews on the code you have written, where the main objective is everyone discussing some things where you (so far) had not had agreements. This may include syntax and alignment, but will more likely include things like just how much do you document, the better testing approaches you'd prefer to normalize on, and so on.
+- when you get stumped, have a quick chat to figure out what approaches are possible, what you will do to experiment and find out, and then figure out how to make a decision
+- do experiment reports where if a few people on your team worked on a thing, they report back on their experience to other people.
+
+You'll also want to ask people for help online. Answers might be slow, but they can eventually come. StackOverflow is alright, but has never really managed to foster a community for Erlang. More interactive areas (slack, IRC, mailing lists, forums) tend to have longer-lasting memberships with a better way to figure out what you actually need. Remember that newcomers to a technology are always at risk of suffering from the [[https://en.wikipedia.org/wiki/XY_problem][XY problem]], and so will your team.
```

Comment:
Should be linking to the chats, lists, forums, etc. Likely link to a separate section that includes this instead of putting them all here because we'll want to reference it in multiple places.